cmd/kubeadm/app/kubeadm.go

```diff
@@ -21,10 +21,10 @@ import (
 
 	"github.com/spf13/pflag"
 
-	_ "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/install"
```

Comment:
This is crucial, and we should have a comment here...
This makes kubeadm able to use the `kubeadm.k8s.io` API group, by installing it via this import...